src/ontology/mondo-edit.obo

```diff
@@ -435463,6 +435463,18 @@ synonym: "attenuated hypophosphatasia" EXACT [https://www.clinicalgenome.org/aff
 is_a: MONDO:0018570 {source="PMID:32973344", source="https://www.clinicalgenome.org/affiliation/40065/"} ! hypophosphatasia
 property_value: created_by http://orcid.org/0000-0002-5460-8025
 
+[Term]
+id: MONDO:0700002
+name: ATP1A3-associated neurological disorder
+def: "Any neurological disorder in which the cause of the disease is a mutation in the ATP1A3." [MONDO:patterns/disease_series_by_gene]
```

Comment:
You can add the PMID references to the definition as well.

Reply:
added PMID